apophis/__main__.py

```diff
@@ -4,11 +4,21 @@
 import discord
 import os
 import time
-
+import logging
 import cmds.command
 from core.readers import fetch_config
 from core.http import http_handler
 
+
+DEBUG = False
```

Comment:
We don't need to track whether debug is enabled with a variable. Check out `logging.setLevel`.